src/richie/apps/core/cache.py

```diff
@@ -131,3 +133,35 @@ def close(self, **kwargs):
         del self._client_read
         self._client_write = None
         self._client_read = None
+
+
+class LimitBrowserCacheTTLHeaders(MiddlewareMixin):
+    """
+    This middleware allows to define a maximum cache timeout for the cache
+    response headers (Cache-control: max-age and Expires).
+
+    It can be configured with the MAX_BROWSER_CACHE_TTL setting.
+    This value is expressed in seconds.
+
+    LimitBrowserCacheTTLHeaders should be placed at the beggining of the
+    MIDDLEWARE list, so that it'll get called last during the response phase.
+    """
+
+    # pylint: disable=no-self-use
+    def process_response(self, request, response):
+        """
+        Rewrite the Cache-control and Expires headers in the response
+        if needed.
+        """
+        max_ttl = getattr(settings, "MAX_BROWSER_CACHE_TTL", None)
+        if max_ttl is None:
+            return response
```

Comment:
Maybe cover also the case of invalid values?
```suggestion
try:
    max_ttl = int(max_ttl)
except Exception as err:
    logger.error(err)
    return response
```

And add a test for this case?